boards/arm/stm32f103_mini/doc/index.rst

```diff
@@ -0,0 +1,162 @@
+.. _stm32f103_mini_board:
+
+STM32F103 Mini
+################
+
+Overview
+********
+
+The STM32F103_MINI board features an ARM Cortex-M3 based STM32F103RC MCU
+with a wide range of connectivity support and configurations. There are
+multiple version of this board like ``stm32f103_mini``.
+
+.. image:: img/stm32f103_mini_yellow.jpg
+   :width: 500px
+   :height: 367px
+   :align: center
+   :alt: STM32F103 Mini Yellow
+
+.. image:: img/stm32f103_mini_blue.jpg
+   :width: 500px
+   :height: 367px
+   :align: center
+   :alt: STM32F103 Mini Blue
+
+Hardware
+********
+STM32F103 Mini provides the following hardware components:
+
+- STM32 microcontroller in QFP64 package
+
+- Flexible board power supply:
+
+  - USB VBUS or external source (3.3V, 5V, 7 - 12V)
+  - Power management access point
+
+- Two LEDs:
+
+  - User LED (LD1), power LED (LD2)
+
+- USB re-enumeration capability:
+
+  - Mass storage
+
+- Support of wide choice of Integrated Development Environments (IDEs) including:
+
+  - IAR
+  - ARM Keil
+  - GCC-based IDEs
```

Comment:
This part is not pertinent for a Zephyr documentation, please remove.

Reply:
Just removed it.